config/credentials/production.yml.enc

```diff
@@ -0,0 +1 @@
+hP5ClNN6t62jeWZOvILFTGYXzwlr6Q+o1uK8ZBVxZaf26Q39/fKBagAiOJu5E23aKAb0HQxi+ewPoGJoStLnFlJ2wWr8mIEMrIk9H243YKwsbji7M2U8HEApWHRyJXzmdqrssaUuuRSQAs6ilue/Fe4G1l4LJaw7EfFW7g==--cT02rktxlmprEp4+--ZuYO6Hq5Z42cABinDmX7pQ==
```

Comment:
Created a new GitHub app, credentials are here.

The production key is in the password database.